docs/src/main/sphinx/release/release-415.md

```diff
@@ -0,0 +1,36 @@
+# Release 415 (25 Apr 2023)
+
+## General
+
+* Improve performance consistency of aggregations with variable file sizes. ({issue}`11361`)
+
+## Web UI
+
+* Add CPU planning time to the query details page. ({issue}`15318`)
+
+## Delta Lake connector
+
+* Add support for commenting on tables and columns with an `id` and `name`
+  column mapping mode. ({issue}`17139`)
+
+## Hive connector
+
+* Improve performance of queries with selective filters on primitive fields in
+  `ROW` columns. ({issue}`15163`)
+
+## Iceberg connector
+
+* Improve performance of queries with filters when Bloom filter indexes are
```

Comment:
This is just new in Iceberg right .. it was there in Hive already from what I recall